department-of-veterans-affairs/va.gov-cms
# VACMS-3370: Updated character limit for meta description field

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3370

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Confirm the 'Meta Description' field has a character limit of **300**.
  - [ ] Navigate here: /node/add/basic_landing_page
  - [ ] Meta Description field has a character limit of **300**.
  - [ ] Save successful.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
